packages/office-ui-fabric-react/src/components/Button/BaseButton.tsx

```diff
@@ -37,6 +40,14 @@ interface IButtonClassNames {
 
 export class BaseButton extends BaseComponent<IBaseButtonProps, IBaseButtonState> implements IButton {
 
+  private get _isSplitButton(): boolean {
+    return (!!this.props.menuProps && !!this.props.onClick);
+  }
+
+  private get _isExpanded(): boolean {
```

Comment:
This is different though. `isExpanded` checks for `this.state.menuProps`, and here we want to see if menuProps have been passed, not if the menu is expanded or not.